bufferserver/src/test/java/com/datatorrent/bufferserver/server/ServerTest.java

```diff
@@ -72,7 +72,8 @@ public static void setupServerAndClients() throws Exception
 
     instance = new Server(0, 4096,8);
     address = instance.run(eventloopServer);
-    assert (address instanceof InetSocketAddress);
+    assertTrue(address instanceof InetSocketAddress);
+    assertFalse(address.isUnresolved());
```

Comment:
I think assert for InetSocketAddress (address is declared as InetSocketAddress) and dealing with unresolved address were parts of the original code that was later refactored. Server.run() returns resolved InetSocketAddress.